### PR TITLE
cleanup stopch(#497)

### LINKED ISSUE #497: Cleanup:  eliminating stopCh defined everywhere
This is the searching result.

https://github.com/search?q=repo%3Akmesh-net%2Fkmesh%20make(chan%20struct%7B%7D)&type=code

By passing stop channel to each module's Start/Run method, we can control each module globally. And even more, by doing so, we could further eliminate `defer stop` stuff
```
	c := controller.NewController(configs, bpfLoader.GetBpfKmeshWorkload(), configs.BpfConfig.BpfFsPath, configs.BpfConfig.EnableBpfLog)
	if err := c.Start(); err != nil {
		return err
	}
	log.Info("controller Start successful")
	defer c.Stop()
```

### PR DESCRIPTION
**Pull request by Okabe-Rintarou-0**
**What type of PR is this?**
enhance

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #497 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: pkg/controller/bypass/bypass_controller.go [hunk 1]
```diff
@@ -48,8 +48,7 @@ const (
 	SidecarAnnotation         = "sidecar.istio.io/inject"
 )
 
-func StartByPassController(client kubernetes.Interface) error {
-	stopChan := make(chan struct{})
+func StartByPassController(client kubernetes.Interface, stopChan <-chan struct{}) error {
```
Comment (LiZhenCheng9527) [inline pkg/controller/bypass/bypass_controller.go:51]: Can you replace stopChan with context.Context

Comment (Okabe-Rintarou-0) [reply pkg/controller/bypass/bypass_controller.go:51]: you mean all of the stopch? I am confused about why there exist both ctx and stopCh.

Comment (Okabe-Rintarou-0) [reply pkg/controller/bypass/bypass_controller.go:51]: https://github.com/kmesh-net/kmesh/blob/ba94cbc1be2650d4a8b38adb1aefa62a764735c6/pkg/controller/bypass/bypass_controller.go#L135

informer needs a stopch

**Review by hzxuzhonghu — COMMENTED** [review]
LGTM, I would do a little bit further, pass in from `func Execute(configs *options.BootstrapConfigs) error {`

File: pkg/controller/bypass/bypass_controller.go (hunk 1 above)
Comment (LiZhenCheng9527) [reply pkg/controller/bypass/bypass_controller.go:51]: Indeed, the informer need

**Review by hzxuzhonghu — APPROVED** [review]
/lgtm

**Pull request merged**